src/main/webapp/mapsScript.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Create the script tag, set the appropriate attributes
+var script = document.createElement('script');
+script.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyCmQyeeWI_cV0yvh1SuXYGoLej3g_D9NbY&libraries=places&callback=initMap';
+script.defer = true;
+script.async = true;
```

Comment:
This code is equivalent to adding a <script> element in the HTML file, which I use to add the Maps API script. I chose to add it in the JS file because I was having issues with loading the API and running the init function asynchronously, and found this to be the solution.

Reply:
+1 I would mildly prefer this to be declared in the HTML file, or perhaps there's a way to directly fetch this URL from the javascript?
No need to change in this CL though. Let me know if you want to debug the issues you ran into together. I may be wrong, and this may be the only way to do it.